data/preprocess_audio.py

```
@@ -0,0 +1,135 @@
+        output_path.parent.mkdir(parents=True, exist_ok=True)
+        sf.write(output_path, audio, target_sr)
+        
+        return True, None
+    except Exception as e:
+        return False, str(e)
+
+
+def preprocess_dataset(
+    input_dir: Path,
+    output_dir: Path,
+    extensions: List[str] = [".wav", ".flac"],
```

Ruff (0.14.13)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)